meta-luneos/recipes-webos/sam/sam/0011-NativeContainer-configure-native-apps.patch

```diff
@@ -0,0 +1,32 @@
+From ca59d2722cb956b568c4444593970f52938f54ae Mon Sep 17 00:00:00 2001
+From: Christophe Chapuis <chris.chapuis@gmail.com>
+Date: Tue, 21 Feb 2023 17:57:20 +0000
+Subject: [PATCH] NativeContainer: configure native apps
+
```

Comment:
No big deal, but please try to use Upstream-Status in future. see 617a7ee19e9f47d993f308264ff9392ddd9a0678 which you didn't have a chance to notice when rebasing this PR, I'll add it now with some other sam changes.